pkg/engine/const.go

```diff
@@ -192,6 +192,7 @@ const (
 	kubernetesParams              = "k8s/kubernetesparams.t"
 	kubernetesWinAgentVars        = "k8s/kuberneteswinagentresourcesvmas.t"
 	kubernetesWinAgentVarsVMSS    = "k8s/kuberneteswinagentresourcesvmss.t"
+	kubernetesParameters          = "k8s/kubernetesparameters.t"
```

Comment:
let's also change this const to reflect that, ultimately, it points to the template file that will represent that the ARM template parameters (de-emphasize the "kubernetes" aspect of it)

Reply:
Done